queen-infra-db/src/main/java/fr/insee/queen/infrastructure/db/data/repository/jpa/DataJpaRepository.java

```diff
@@ -1,21 +1,18 @@
-package fr.insee.queen.infrastructure.db.surveyunit.repository.jpa;
+package fr.insee.queen.infrastructure.db.data.repository.jpa;
 
 import com.fasterxml.jackson.databind.node.ObjectNode;
-import fr.insee.queen.infrastructure.db.surveyunit.entity.DataDB;
+import fr.insee.queen.infrastructure.db.data.entity.common.DataDB;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Modifying;
 import org.springframework.data.jpa.repository.Query;
-import org.springframework.stereotype.Repository;
+import org.springframework.data.repository.NoRepositoryBean;
 import org.springframework.transaction.annotation.Transactional;
 
 import java.util.Optional;
 import java.util.UUID;
 
-/**
- * JPA repository to handle survey unit's data response for a questionnaire
- */
-@Repository
-public interface DataJpaRepository extends JpaRepository<DataDB, UUID> {
+@NoRepositoryBean
```

Comment:
a comment here could help to understand this interface provide default methods, and classes implementing it only handle cyphered/notCyphered cases